### PR TITLE
Python changelog and 0.1.3b16 release

### PR DESCRIPTION
**Pull request by joeldodge79**
Started a changelog beginning with the newest release and updated the
readme.

### CODE REVIEW

File: python/README.rst [hunk 1]
```diff
@@ -21,10 +21,19 @@ Basic Usage
...
+    print(my_user["first_name"])
+    # or a model instance (User in this case)
+    print(my_user.first_name)
+
+    # input methods can take either model instances like WriteUser
+    sdk.create_user(
+        body=looker_sdk.models.WriteUser(first_name="Jane", last_name="Doe")
+    )
+    # or plain dictionaries
+    sdk.create_user(body={"first_name": "Jane", last_name: "Doe"})
```
Comment (jkaster) [inline python/README.rst:36]: Do you want to cover how extra properties are ignored?

**Review by jkaster — APPROVED** [review]
This looks good. My question is not essential for this PR.

File: python/README.rst (hunk 1 above)
Comment (joeldodge79) [reply python/README.rst:36]: I want to keep it super brief for this basic usage. probably time to add some examples (both for dictionary and for enums)

**Review by josephaxisa — APPROVED** [review]
Nice!

**Comment by joeldodge79** [comment]
actually, now that sdk examples is in this repo I'll make some changes there to include in this PR

**Comment by jkaster** [comment]
> actually, now that sdk examples is in this repo I'll make some changes there to include in this PR

Glad our migration of examples turns out to useful in exactly a way we anticipated!

**Comment by joeldodge79** [comment]
@jkaster and @josephaxisa I added an example that demonstrates the new dictionary usage as well as Enums and tested it against the new package (using the `testpypi` source). We should be good to go.

**Review by jkaster — APPROVED** [review]
LGTM

**Pull request merged**